corehq/apps/app_manager/suite_xml/case_tile_templates/icon_text_grid.xml

```diff
@@ -0,0 +1,182 @@
+<detail id="{detail_id}">
+  <title>
+    <text>
+      <locale id="{title_text_id}"/>
+    </text>
+  </title>
+
+  <field>
+    <style horz-align="center" vert-align="top" font-size="small">
```

Comment:
The image fields all have a `vert-align` top here, but center in the json template. I think that should be updated in one place or the other.